website/content/api-docs/system/rate-limit-quotas.mdx

```diff
@@ -129,7 +134,9 @@ $ curl \
 
 ## List rate limit quotas
 
-This endpoint returns a list of all the rate limit quotas.
+This endpoint returns a list of all the lease count quotas across all namespaces.
```

Comment:
**NOTE:** This is a typo and should be "rate limit" instead of "lease count." I unfortunately did not catch it until after merging. A new https://github.com/hashicorp/vault/pull/25899 has been opened to correct it on main. Once merged, it will be backported accordingly.